install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Based on Deno installer: Copyright 2019 the Deno authors. All rights reserved. MIT license.
+
+set -e
+
+case $(uname -sm) in
+"Darwin x86_64") target="Darwin_x86_64" ;;
+"Darwin arm64") target="Darwin_arm64" ;;
+"Linux aarch64") target="Linux_arm64" ;;
+*) target="Linux_i386" ;;
+esac
+
+version=${1}
+
+if [ $# -eq 0 ]; then
+    version="$(curl -s https://api.github.com/repos/thegreenwebfoundation/grid-intensity-go/releases/latest | grep tag_name | cut -d '"' -f 4)"
+fi
+
+version="$(echo $version | cut -d 'v' -f 2)"
+grid_intensity_uri="https://github.com/thegreenwebfoundation/grid-intensity-go/releases/download/v${version}/grid-intensity_${version}_${target}.tar.gz"
```

Comment:
This mostly looks good to me. For shellscripts, it's worth running content through https://www.shellcheck.net to catch any common linting errors. Here's the output I got when running this file through it.

```
[Line 19:](javascript:setPosition(19, 17))
version="$(echo $version | cut -d 'v' -f 2)"
                ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([0])), apply [all SC2086](javascript:applyFixCode(2086)))
version="$(echo "$version" | cut -d 'v' -f 2)"
```

Reply:
Thanks @mrchrisadams great idea to run it through the linter and that's fixed now.